daemons/gptp/gptp_cfg.ini

```diff
@@ -31,19 +31,11 @@ syncReceiptThresh = 8
 
 [eth]
 
-# PHY delay GB TX in nanoseconds
+# PHY delay GB TX/RX in nanoseconds
```

Comment:
I can back this out for now, but I don't think it's scalable to have fixed configuration options like this. For wireless, it'll be awkward. Eventually I think we should have something like phy_delay 1000000 384 184. What I proposed doesn't do this but it's part way.
I suppose we could have both?